### ISSUE TITLE
3.7 feedback

### ISSUE THREAD

**Issue by eoiz** [akai-katto/dandere2x#270]
I tried a bit the new version and it IS faster (same input, same config: 3.5 1,45m; 3.6 2,45m; 3.7 1,29m).
The problem is that it doesn`t work with long videos.
I tried upscaling 5 anime episodes and no one gets upscaled past 2-3 minutes...just after the end of the second chapter.
[37.txt](https://github.com/akai-katto/dandere2x/files/9883506/37.txt)

So I tried a video with no chapter and it worked...almost. It has upscaled 13m 54s and left 32s as a static image (with audio playing correctly) 
[37 14m.txt](https://github.com/akai-katto/dandere2x/files/9883503/37.14m.txt)

**Comment by akai-katto** [comment]
Can yous end me the video over reddit?

**Comment by eoiz** [comment]
First https://drive.google.com/file/d/1fx3_Tqeh1bSfLFOpsKYlEPsdmpqxYfmd/view?usp=sharing
Second https://drive.google.com/file/d/1fx3_Tqeh1bSfLFOpsKYlEPsdmpqxYfmd/view?usp=sharing

**Comment by akai-katto** [comment]
I meant the upscale videos, the originals so I replicate the error

**Comment by eoiz** [comment]
Oh sorry, here:
Anime* https://drive.google.com/file/d/19k6E2nVwrqjkFAJBh6Lk8HWEM96KkHLt/view?usp=sharing
No chapter* https://drive.google.com/file/d/15XP0VvRxxwbXQbb1w9YKQSFY_2QrSVJR/view?usp=sharing

**Comment by akai-katto** [comment]
Whats your pc specs, btw? I cant remember if I asked in a previous thread or not

**Comment by eoiz** [comment]
Windows 11 with an i9 13900k, z690 carbon ek, rtx 3080 ti (msi suprim x), 64gb ddr5@4GHz (I run Dandere2x off from a ram disk) everything under custom water loop with a 360 and a 240 radiators.
Used to be a 12900KS but with thermal problems: very very glad I buyed this one instead, max temp I've seen is 92C.
BTW thank you very much for the frame ahead thing =)
Also, are you all right now? I read about a medical emercency...

**Comment by akai-katto** [comment]
replicable on my spare pc, which is good news.

**Comment by akai-katto** [comment]
https://github.com/akai-katto/dandere2x/releases/tag/3.7

Try this new patch

**Comment by eoiz** [comment]
Yes that works, but it has become fast as the 3,5 version that has no other bugs...

**Comment by akai-katto** [comment]
What do you mean? Does 3.7.2 work as intended? I don't follow.

**Comment by eoiz** [comment]
The patch works as intended, but brings compute time same as version 3.5 (1,45m for upscaling the sample video).
But until the 3.7.2 patch comes I cannot try longer videos or multiprocess...so...will wait :)

**Comment by akai-katto** [comment]
3.7.2 is already posted? It should be found here:

https://github.com/akai-katto/dandere2x/releases/tag/3.7

Right below 3.7.1

**Comment by eoiz** [comment]
Sorry I totally missed the patch!

So, now I've installed it and upscaled a video:
[372 multi.txt](https://github.com/akai-katto/dandere2x/files/9906368/372.multi.txt)
I used the multiprocess option but the audio goes offsync at some point (not from the start): the original video has 23.976 fps while the upscaled one has 24.417 (with MKVToolNix you can force it and will play correctly).

Also the output file is h264 (4823kbps) while the configuration file in set to h265.
The 3 splitted video in the gui folder is actually h265 (7645kbps), but the final output is h264.

Other than this it works and seems pretty fast!

**Comment by eoiz** [comment]
Ok, after upscaling more than 20 anime episodes I encountered another problem:
upscaling this (https://drive.google.com/file/d/19k6E2nVwrqjkFAJBh6Lk8HWEM96KkHLt/view?usp=share_link) the output video freezes at the end of chapter 7 (quick fix will be upscaling every chapter separately and then remux all together, but still...).
